### PR TITLE
Improve resilience on handling sharing attributes

### PR DESCRIPTION
**Pull request by guerler**

### CODE REVIEW

**Comment by martenson** [comment]
is this broken for some items? do we need to backport?

**Comment by guerler** [comment]
It caused issues with sharing visualizations in dev.

File: lib/galaxy/web/base/controller.py [hunk 1]
```diff
@@ -1407,17 +1407,17 @@ def sharing(self, trans, id, payload=None, **kwd):
             action = payload.get("action")
             if action == "make_accessible_via_link":
                 self._make_item_accessible(trans.sa_session, item)
-                if item.has_possible_members and payload.get("make_members_public", False):
+                if hasattr(item, "has_possible_members") and payload.get("make_members_public", False):
```
Comment (martenson) [inline lib/galaxy/web/base/controller.py]:
```suggestion
                if hasattr(item, "has_possible_members"):
                    if item.has_possible_members and payload.get("make_members_public", False):
```

Comment (martenson) [reply lib/galaxy/web/base/controller.py]: Trying new github feature here. :)

The point is that it should still check the truthiness of the attribute if it exists.

Comment (guerler) [reply lib/galaxy/web/base/controller.py]: Makes sense.

**Comment by guerler** [comment]
@galaxybot test this.

**Pull request merged**